zerver/lib/import_realm.py

```diff
@@ -1103,6 +1107,15 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
     if not os.path.exists(import_dir):
         raise Exception("Missing import directory!")
 
+    migration_status_filename = os.path.join(import_dir, "migration_status.txt")
+    if os.path.exists(migration_status_filename):
+        # Ignore export bundle without migration_status.txt to support
+        # backward compatibility.
```

Comment:
I don't think we need to do this term; we only support import/export from the same version. But I think you do need to add code in the `zerver/data_export` tools to generate the migration status file too, or those will crash after adding the requirement that a migration status file is present.

Reply:
>... we only support import/export from the same version.

Arg., right. Now that you mention it, this check defeats the purpose of the PR :sweat_smile:  

I'll add another commit to adjust the codes in `zerver/data_import`